### ISSUE TITLE
Unable to connect to eu.posthog.com

### ISSUE THREAD

**Issue by g-hamilton** [PostHog/posthog#21280]
Hi,

I have a cloud project in the EU data region. I'm experiencing constant connectivity problems with eu.posthog.com. I've tried on Chrome and Safari using a superfast network connection and I get a white loading screen if I try to access http://eu.posthog.com/project/{myprojectid}, which eventually becomes an error screen saying the website took too long to respond. I get the same issue if I visit http://eu.posthog.com/signup.

It seems sporadic as well. Earlier today I was able to load my cloud project with no issue, but after a few minutes I got a loading state in my project home which did not complete, and when I tried to refresh I was unable to connect again.

I've tried clearing my browser cache and cookies, restarting browser, different browser, different device on same network, clearing CDN cache on my mac, restarting system, disabling firewall and antivirus software and chrome extensions. Nothing works. It's completely isolated to just the eu subdomain. I can access https://posthog.com with no issues, but if I click "Get Started" at the top right of the screen I get the issue again.

I tried logging into my account on my phone. At first I had no issue connecting and then after logging in and viewing my cloud project home screen, after a minute or so the app became unresponsive. On refreshing, same issue - unable to connect - timeout due to taking too long to respond.

If I run `ping eu.posthog.com` in terminal, I get multiple rows of `Request timeout for icmp_seq 0`.

No idea what to try next. I'm thinking about dropping PostHog over this, which seems a shame. Any idea what I can do to resolve this? Is it a CDN issue? Anyone else in the EU region having a problem?

Thanks

**Comment by ItayElgazar** [comment]
I am in the EU Region and can't see any events, at first I thought it's something I did, because I've been touching the events processing in the frontend - but there is an incident (I think so?), you can see it in their status page

**Comment by MarconLP** [comment]
> I am in the EU Region and can't see any events, at first I thought it's something I did, because I've been touching the events processing in the frontend - but there is an incident (I think so?), you can see it in their status page

We are currently experiencing an ingestion delay of up to 45 minutes. Feel free to follow along on our [status page](https://status.posthog.com/incidents/5rst1wpzkzbq).

**Comment by g-hamilton** [comment]
> > I am in the EU Region and can't see any events, at first I thought it's something I did, because I've been touching the events processing in the frontend - but there is an incident (I think so?), you can see it in their status page
> 
> 
> 
> We are currently experiencing an ingestion delay of up to 45 minutes. Feel free to follow along on our [status page](https://status.posthog.com/incidents/5rst1wpzkzbq).

I did check the status page but that only mentions the US cloud region if I recall correctly. Also, that only mentions a data delay. I can't even load my project or a page that doesn't require authentication, such as https://eu.posthog.com/signup

**Comment by ItayElgazar** [comment]
> > I am in the EU Region and can't see any events, at first I thought it's something I did, because I've been touching the events processing in the frontend - but there is an incident (I think so?), you can see it in their status page
> 
> We are currently experiencing an ingestion delay of up to 45 minutes. Feel free to follow along on our [status page](https://status.posthog.com/incidents/5rst1wpzkzbq).

Yes, though I did check again and @g-hamilton is right - it only mentions the US cloud region. Is the EU affected too?

**Comment by g-hamilton** [comment]
Thanks @MarconLP 

I had a feeling you were going to say you couldn't replicate, as if it was a widespread issue I suspect it would be widely reported. 

What if it is just a problem with my account/project. That might explain why at first I had no issue. It was only after logging in that I started experiencing the issue. I replicated it on my second mobile device in the same way.

Are you able to offer any kind of support? Do you have an ability to login to client accounts? Here's my project ID:

https://eu.posthog.com/project/19542

**Comment by MarconLP** [comment]
> Are you able to offer any kind of support? Do you have an ability to login to client accounts? Here's my project ID. The image shows my browser trying to access this url. Does not load.

I checked your account and was able to successfully check out the project. Do you mind sharing your location?

**Comment by g-hamilton** [comment]
> > Are you able to offer any kind of support? Do you have an ability to login to client accounts? Here's my project ID. The image shows my browser trying to access this url. Does not load.
> 
> 
> 
> I checked your account and was able to successfully check out the project. Do you mind sharing your location?

Sure. It's Cornwall, England, UK

**Comment by g-hamilton** [comment]
Just a follow up to say I experienced the same issue today. At first, I logged into my cloud account and viewed the project with no issues. Everything appeared to be working normally. I fired up my application and sent a couple of events to my project. My project started experiencing issues. I started seeing loading states in the UI and the project stopped responding. If I was to guess, I would say that sending new data/events to the project causes the front-end to stop responding. After a long period of no events, such as overnight, the project 'recovers' and responds normally. It appears the events are recorded correctly as I see no issue with my data.

After sending a few events (or possibly just after a period of time because I'm not sure if sending new events triggers the issue), the UI starts showing loading state permanently:

![image](https://github.com/PostHog/posthog/assets/56267983/0f5fea5d-63e5-4e69-8018-fe51d25d6f6e)

I can initially navigate, but the UI shows loading all over:

![image](https://github.com/PostHog/posthog/assets/56267983/d6173323-8e62-42a3-b749-6cb798b8d922)

Then I get this message:

![image](https://github.com/PostHog/posthog/assets/56267983/a4d2a22d-21ed-4e9c-aee0-ddc5fb821613)
![image](https://github.com/PostHog/posthog/assets/56267983/15d9842e-32a6-4800-91d6-9b58acabf56f)

If I refresh the page, I cannot access the project anymore and the browser returns an error: "The site is taking too long to load".

I have had this behaviour for 3 consecutive days now and I can replicate every day so this is not a one-off issue. I don't think this is a browser, network or system issue on my side. I think this is an issue with PostHog. I notice the issue mentioned above in your status page is now resolved, so it looks like this is a separate and ongoing problem for me.

Should I close my account and signup again?

**Comment by joepavitt** [comment]
We're consistently seeing memory problems on PostHog EU - it's very frustrating. Any Dashbaord we try to use, generally results in a fair few:

<img width="1450" alt="Screenshot 2024-04-03 at 13 04 23" src="https://github.com/PostHog/posthog/assets/99246719/8c46eb27-ae83-4899-b165-57fd3dafea59">

Makes it unusable, and trust in the product has dropped considerably over the past couple of months. Had raised support tickets, but by the time the person addresses it, the reports have loaded, and they're disregarded. Go back a few days later to use PH again, and the errors re-occur.

**Comment by ItayElgazar** [comment]
I second the part about the ingestion issues.

When I report an event that is not auto captured, it doesn't report it until the next event occurs.
